subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -103,13 +103,32 @@ config NRF_CLOUD_MQTT_PAYLOAD_BUFFER_LEN
 	int "Size of the buffer for MQTT PUBLISH payload."
 	default 2048
 
+menu "nRF Cloud FOTA"
+config NRF_CLOUD_FOTA
+	bool "Enable FOTA through nRF Cloud"
+	select FOTA_DOWNLOAD
+	select FOTA_DOWNLOAD_PROGRESS_EVT
+	select DFU_TARGET
+	select DOWNLOAD_CLIENT
+	depends on SETTINGS
+	default y
+
+if NRF_CLOUD_FOTA
 config NRF_CLOUD_FOTA_PROGRESS_PCT_INCREMENT
 	int "Percentage increment at which FOTA download progress is reported"
-	depends on FOTA_DOWNLOAD_PROGRESS_EVT
+	depends on NRF_CLOUD_FOTA
```

Comment:
```suggestion
	depends on NRF_CLOUD_FOTA || FOTA_DOWNLOAD_PROGRESS_EVT

Reply:
If the pure AWS Jobs solution is completely removed, then it won't be needed. If it's not removed then this is needed for having the possibility of enabling it.